### PR TITLE
css refactor so that elements don't overlap

### PR DESCRIPTION
**Pull request by tillywoodfield**

### CODE REVIEW

File: src/components/Dashboard/Dashboard.module.css [hunk 1]
```diff
@@ -1,15 +1,46 @@
...
+.logo {
+    flex-grow: 1;
+    padding: 1em;
+}
+
+.main {
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+
```
Comment (sibela46) [inline src/components/Dashboard/Dashboard.module.css]: extra space

File: src/components/Lobby/Lobby.jsx [hunk 2]
```diff
@@ -34,10 +34,10 @@ export class Lobby extends React.Component {
 
     render() {
         return (
-            <div className={styles.lobbyPage}>
+            <div>
```
Comment (sibela46) [inline src/components/Lobby/Lobby.jsx]: Maybe put some className in that div, or if it's not gonna have any just wrap the return in <> </>?

File: src/components/Logout/Logout.jsx [hunk 3]
```diff
@@ -2,12 +2,11 @@ import React from 'react';
...
 import { connect } from 'react-redux';
 import { logOutUser } from '../shared/Auth/AuthActions';
-import styles from './Logout.module.css';
 
 export class Logout extends React.Component {
     render() {
         return (
-            <div className={styles.logoutButton}>
+            <div>
                 <Button cy="logout" handleClick={this.props.logOutUser} text="Log out" />
```
Comment (sibela46) [inline src/components/Logout/Logout.jsx]: Why does this need to be wrapped in a div

File: src/components/shared/CardList/Card/Card.jsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ export class Card extends React.Component {
 
     render() {
         return (
-            <div data-cy={`card-wrapper-${this.props.enabled ? 'enabled' : 'disabled'}`} className={cx(styles.cardWrapper, { enabled: this.props.enabled }, { disabled: !this.props.enabled }, {selected : this.props.playedCardId === this.props.card.cardId})} onClick={this.handleClick}>
+            <div data-cy={'card-wrapper'} className={cx(styles.cardWrapper, { enabled: this.props.enabled }, { disabled: !this.props.enabled }, {selected : this.props.playedCardId === this.props.card.cardId})} onClick={this.handleClick}>
```
Comment (sibela46) [inline src/components/shared/CardList/Card/Card.jsx]: no curly braces needed

**Review by sibela46 — APPROVED** [review]

File: src/components/Lobby/Lobby.jsx [hunk 5]
```diff
@@ -34,14 +34,14 @@ export class Lobby extends React.Component {
 
     render() {
         return (
-            <div className={styles.lobbyPage}>
+            <>
```
Comment (scottlogic-alex) [inline src/components/Lobby/Lobby.jsx:37]: is this nameless tag intentional?

Comment (sibela46) [reply src/components/Lobby/Lobby.jsx:37]: yeah, react complains when you return multiple tags, they need to be wrapped in something

File: src/components/StartGame/StartGame.jsx [hunk 6]
```diff
@@ -15,7 +15,9 @@ export class StartGame extends React.Component {
 
     render() {
         return (
-            <Button cy="start-game" handleClick={this.startGame} text="Start game" />
+            <div>
```
Comment (scottlogic-alex) [inline src/components/StartGame/StartGame.jsx:18]: why wrap in a div? are you just trying to make it `display: block` ?

Comment (sibela46) [reply src/components/StartGame/StartGame.jsx:18]: that's outdated

**Pull request merged**